leakcanary-android-core/src/main/java/leakcanary/internal/AndroidHeapDumper.kt

```diff
@@ -101,12 +101,12 @@ internal class AndroidHeapDumper(
       toast.view = inflater.inflate(R.layout.leak_canary_heap_dump_toast, null)
       toast.show()
 
-      val toastIcon = toast.view.findViewById<View>(R.id.leak_canary_toast_icon)
-      toastIcon.translationY = -iconSize.toFloat()
+      val toastIcon = toast.view?.findViewById<View>(R.id.leak_canary_toast_icon)
```

Comment:
This should be `!!` here, and the rest shouldn't change. `Toast.getView()` returns null if the toast was created using Toast.makeText(), which isn't the case here.

https://developer.android.com/reference/android/widget/Toast

Reply:
Done